django/test/client.py

```diff
@@ -701,7 +701,7 @@ def _parse_json(self, response, **extra):
         return response._json
 
     def _handle_redirects(self, response, **extra):
-        "Follows any redirects by requesting responses from the server using GET."
+        "Follow any redirects by requesting responses from the server using GET."
```

Comment:
Pep257 says, "Triple quotes are used even though the string fits on one line. This makes it easy to later expand it."